src/riak_kv_pb_timeseries.erl

```diff
@@ -120,7 +122,7 @@ encode(Message) ->
     {ok, riak_pb_codec:encode(Message)}.
 
 
--spec process(atom() | #tsputreq{} | #tsdelreq{} | #tsgetreq{} | #tslistkeysreq{}
+-spec process(atom() | #tsputreq{} | #tsputreqttb{} | #tsdelreq{} | #tsgetreq{} | #tslistkeysreq{}
```

Comment:
So I did see your commit to change this name to avoid collision with previous versions, but the new name isn't terribly consistent with all of the others.  Do we actually have customers in the wild using the **TsTtbPutReq** message?